engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -131,19 +131,6 @@ namespace dmGameSystem
     static float GetPlaybackRate(SpriteComponent* component);
     static void SetPlaybackRate(SpriteComponent* component, float playback_rate);
 
-    template<typename T>
-    void fillIndices(T* index, uint32_t indices_count) {
-        for(uint32_t i = 0, v = 0; i < indices_count; i += 6, v += 4)
-        {
-            *index++ = v+0;
-            *index++ = v+1;
-            *index++ = v+2;
-            *index++ = v+2;
-            *index++ = v+3;
-            *index++ = v+0;
-        }
-    }
-
```

Comment:
This is written when looping through the sprites now, since the vertex pattern for slice-9 is a little bit different. We generate 16 vertices in a grid and re-use these vertices instead of creating 9 unique quads.